addons/website/views/snippets/snippets.xml

```diff
@@ -934,12 +934,22 @@
     <!-- Anchor Name -->
     <div data-js="anchor"
         data-selector=":not(p).oe_structure > *, :not(p)[data-oe-type=html] > *"
-        data-exclude=".modal *, .oe_structure .oe_structure *, [data-oe-type=html] .oe_structure *">
+        data-exclude=".modal *, .oe_structure .oe_structure *, [data-oe-type=html] .oe_structure *, .s_popup, .o_newsletter_popup">
         <we-button class="fa fa-fw fa-link o_we_link"
                    title="Create a link to target this section"
                    data-no-preview="true"/>
     </div>
 
+    <div data-js="anchor"
+        data-selector=".s_popup, .o_newsletter_popup"
+        data-target=".modal">
+        <div class="d-none">
```

Comment:
I'd prefer a `ModalAnchor` JS option as an extension of the base anchor option and with `_computeVisiblity` overridden to not be shown, with an explanation in there.

Actually, even better: not create an element you don't use: you can use the notify system. Instead of finding the widget and simulate a click on it, trigger_up the "option_update" event and ask the anchor option to do the job

Reply:
With ClipboardJS, it's not possible de copy the link in the clipboard without using a click event (see [here](https://stackoverflow.com/questions/43235239/copy-text-to-clipboard-using-clipboard-js-without-button)
So are you okay if I don't use "option_update" and use your first solution